client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -21,6 +22,7 @@ export default connectOptions(
 
 		return (
 			<ThemeShowcase { ...props } siteId={ siteId }>
```

Comment:
Is the order correct for `QuerySitePlans` ? Will not render hit `themes-selection` component before `QuerySitePlans`?